### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 28

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#354] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152325993-e36fd392-491a-4832-8a24-86ac256aea81.png)

**Comment by ForzaNapolixsempre** [comment]
![gdgdg](https://user-images.githubusercontent.com/122674373/212470322-428d836a-cc6b-40bf-adbd-1780acb56eea.jpg)
da come ho capito non puoi eliminare due lettere insieme per questo penso che il ragionamento del collega Elia sia errato, correggetemi se sbaglio.
